makefile

```diff
@@ -1,5 +1,11 @@
 default:
-	cake -target=default
+		cake -target=default
 
 deps:
-	cake -target=deps
+		cake -target=deps
+
+dependencies:
```

Comment:
You should really have a `.PHONY` rule that lists these targets, so make doesn't think these rules produce a file of the target name.